subiquitycore/models/network.py

```diff
@@ -228,6 +231,12 @@ def ip6_providers(self):
         ip_info = self._get_ip_info()
         return ip_info['ip6_providers']
 
+    def is_connected(self):
```

Comment:
I think this properly belongs in probert rather than directly in our models for subiquity.